src/XrdCeph/XrdCephBulkAioRead.hh

```diff
@@ -0,0 +1,50 @@
+#include <string>
+#include <map>
+#include <list>
+#include <tuple>
+#include <rados/librados.hpp>
+
+typedef std::tuple<ceph::bufferlist*, char*, int*> ReadOpData;
+typedef void (*logfunc_pointer) (char *, ...);
```

Comment:
Are these better declared inside bulkAioRead?

Reply:
Yes, they are. Moved to class